app/javascript/components/rooms/room/FeatureTabs.jsx

```diff
@@ -27,17 +31,6 @@ export default function FeatureTabs({ shared }) {
     );
   }
 
-  // Returns only the Recording tab if the room is a Shared Room and the User does not have the ManageRoom permission
-  if (shared && !currentUser?.permissions?.ManageRooms) {
-    return (
-      <Tabs className="wide-white pt-4 mx-0" defaultActiveKey="recordings" unmountOnExit>
-        <Tab className="background-whitesmoke" eventKey="recordings" title={t('recording.recording')}>
-          <RoomRecordings />
-        </Tab>
-      </Tabs>
-    );
-  }
```

Comment:
This has never worked, related to the comment above.
Passed down props was not properly dealt with - it needs a placeholder/isLoading.
It was set by default as 'false' and would not re-render for some reason.